sdk/communication/communication-callingserver/src/callingServerClient.ts

```diff
@@ -824,4 +830,81 @@ export class CallingServerClient {
       span.end();
     }
   }
+
+    /**
+   * Deletes the content pointed to the uri passed as a parameter.
+   *
+   * * In Node.js, data returns ?
+   * * In browsers, data returns ?
+   *
+   * @param deleteUri - Endpoint where the content exists.
+   *
+   * Example usage (Node.js):
+   *
+   * ```js
+   * // Delete and convert a blob to a string
+   * const deleteUri = "https://deleteUri.com";
+   * const deleteResponse = await callingServerClient.delete(deleteUri);
+   *
+   * ```
+   */
+  public async delete(
+    deleteUri: string,
+    options: OperationOptions = {}
+  ): Promise<RestResponse>
+  {
+    const { span, updatedOptions } = createSpan("ServerCallRestClient-delete", options);
+
+    const operationArguments: OperationArguments = {
+      options: operationOptionsToRequestOptionsBase(updatedOptions)
+    };
+
+    try 
+    {
+      const q = URLBuilder.parse(deleteUri);
+      const formattedUrl = q.getPath()!.startsWith("/") ? q.getPath()!.substr(1) : q.getPath()!;
+      const stringToSign = this.deleteAndDownloadCallingServerApiClient.endpoint + formattedUrl;
```

Comment:
I don't fully remember, but aren't these the exact same lines as in download? Could we refactor and clean this?